### ISSUE TITLE
Heroku CI fails

### ISSUE THREAD

**Issue by alessandromacagno** [jonathanong/heroku-buildpack-ffmpeg-latest#50]
Hello @jonathanong,

thanks a lot for this repo, it worked perfectly until this morning.

Now, in Heroku CI, we have a random failure. Do you have any idea how to fix it? Thanks in advance
```

       Downloading https://johnvansickle.com/ffmpeg/builds/ffmpeg-git-amd64-static.tar.xz
.       Unpacking the archive
xz: (stdin): Unexpected end of input
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now
       Failed to unpack
```

**Comment by barkovv** [comment]
I have the same issue, manual triggering of new deploy helps (it seems that the johnvansickle.com host is unstable). 

Probably it's a good idea to fallback to https://github.com/eugeneware/ffmpeg-static/releases (Github site is more reliable )

**Comment by barkovv** [comment]
Here is the link for latest x64 file:
https://github.com/eugeneware/ffmpeg-static/releases/latest/download/linux-x64.gz

**Comment by timlewis** [comment]
Thanks for the link barkovv, unfortunately that file wont work with the buildpack as it uses the command `tar xJf "./$FFMPEG_ARCHIVE_NAME" --strip-components=1` to extract the file and this wont work with a gzipped file

**Comment by farooqch11** [comment]
Hello guys facing same issue with the heroku build pack.   did you guys found any solution?

waiting for this fix, to deploy new releases

**Comment by henriiqueg** [comment]
I'm trying to work on something. Tried to fork this repo and tweak something using the repo that @barkovv sent. But didn't have any progress right now

**Comment by leandroalemao** [comment]
Had the same issue but just tried to use this fork as buildpack and worked well https://github.com/hmh84/heroku-buildpack-ffmpeg-latest.git

**Comment by farooqch11** [comment]
@leandroalemao  you mean run this command  to add this as buildpack for heroku  or just change url in the ENV?

heroku create --buildpack  https://github.com/hmh84/heroku-buildpack-ffmpeg-latest.git

**Comment by farooqch11** [comment]
`heroku buildpacks:add --index 1 https://github.com/hmh84/heroku-buildpack-ffmpeg-latest.git --app universal-promote-dash
`
add this buildpack , worked  fine. thanks @leandroalemao

**Comment by henriiqueg** [comment]
Worked for me @leandroalemao thanks

**Comment by Harsh-br0** [comment]
> Had the same issue but just tried to use this fork as buildpack and worked well https://github.com/hmh84/heroku-buildpack-ffmpeg-latest.git

This too seems temp fix cuz ithat too depends on the site which hosts the bin
Needs something that fetches the latest binary from trusted sources which will not down unexpectedly like github or gitlab

**Comment by leandroalemao** [comment]
> > Had the same issue but just tried to use this fork as buildpack and worked well https://github.com/hmh84/heroku-buildpack-ffmpeg-latest.git
> 
> This seems temp fix cuz that too depends on the site which hosts the bin
> Needs something that fetches the latest binary from trusted sources like github or gitlab which will not down unexpectedly

Agree.. it's just a temporary fix.. I had the whole company waiting for my deployment so it was a quick fix.. but for sure the binary file needs to be hosted in a trusted server.

**Comment by Maldradrem** [comment]
> 
> 
> Had the same issue but just tried to use this fork as buildpack and worked well https://github.com/hmh84/heroku-buildpack-ffmpeg-latest.git

Thanks, it works for now

**Comment by jonathanong** [comment]
https://github.com/jonathanong/heroku-buildpack-ffmpeg-latest/issues/34 if anyone wants to work on it. I can't grant access to this repo too - I don't use this anymore

**Comment by Harsh-br0** [comment]
@jonathanong that doesn't seems to be possible ig , cuz actions triggers on some event like push or pull request , they can't be like cronjobs 
Or maybe peeps can have workaround for that but that's from side as per actions documentations

**Comment by TrisKnightGamer** [comment]
@jonathanong can I get access to this repo? Thank you.
